### ISSUE TITLE
The method 'created_at' was not found on the presenter class

### ISSUE THREAD

**Issue by Mx13Cyber** [mybb/mybb2#238]
for now i have this error
ErrorException in list.twig line 25:
An exception has been thrown during the rendering of a template ("The method 'created_at' was not found on the presenter class 'MyBB\Core\Presenters\UserPresenter'.") in "C:\Wamp\www\mybb2\mybb2\resources\views/member/list.twig" at line 25.

**Comment by euantorano** [comment]
Hm, I'll look into this tomorrow.

**Comment by JordanMussi** [comment]
Tomorrow was a long time ago now :wink:

**Comment by Matslom** [comment]
I explained this bug here https://github.com/mybb/mybb2/pull/253#issuecomment-244076408

**Comment by euantorano** [comment]
Yeah, as @Matslom said, it seems to be caused by null values. The values should never be null, so the seeds need updating to set the `created_at`, `updated_at` and `last_visit` values.
